dev/RadioButtons/RadioButtons.idl

```diff
@@ -14,8 +14,8 @@ unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
     Windows.Foundation.Collections.IVector<Object> Items{ get; };
     Windows.UI.Xaml.DataTemplate ItemTemplate;
 
-    Windows.UI.Xaml.DependencyObject ContainerFromItem(Object item);
-    Windows.UI.Xaml.DependencyObject ContainerFromIndex(Int32 index);
+    Windows.UI.Xaml.UIElement ContainerFromItem(Object item);
+    Windows.UI.Xaml.UIElement ContainerFromIndex(Int32 index);
```

Comment:
Question for PM. MaximumColumns is layout specific. Now that RadioButtons is backed by an ItemsRepeater, you could allow passing a Layout to customize the way the RadioButtons are laid out. Having MaximumColumns on RadioButtons couples layout with RadioButtons. We could instead expose a Layout property on RadioButtons and the default layout set would be experience we want out of the box. MaximumColumns would be a property on the layout.

Reply:
I like this as well.

---
In reply to: [347400724](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347400724) [](ancestors = 347400724)